packages/headless/src/modules/cart/middleware/cart.middleware.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable, NestMiddleware } from '@nestjs/common';
+import { Request, Response, NextFunction } from 'express';
+
+@Injectable()
+export class CartMiddleware implements NestMiddleware {
+  use(req: Request, res: Response, next: NextFunction) {
+    console.log(
+      'Using local middleware:: Requested url:',
+      req.baseUrl,
+      'method:',
+      req.method,
+    );
+    next();
```

Comment:
removed unnecessary logging middleware. I've added as example but not recommended to use everywhere